onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py

```diff
@@ -425,6 +425,11 @@ def _where_input_wrangler(
         matcher=lambda sample: sample.input[0].equal(torch.tensor([])),
         reason="cat does not support zero-dim tensors yet",
     ),
+    # skip(
```

Comment:
You may refer to https://github.com/microsoft/onnx-script/blob/72da830c0e85d8afa1c94f1af8b7c4876090b38b/onnxscript/function_libs/torch_aten/ops/core.py#L2841 to see how we support scalar (tldr check rank and return)

Reply:
Nice